case_study/memory_leaks/leaking_counter_1/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 51;
+	objectVersion = 54;
```

Comment:
accidental changes to this file?

Reply:
This one is needed change: this case study is outdated and needed to test the diff.